bundles/org.eclipse.swt/Readme.md

```diff
@@ -3,12 +3,32 @@ org.eclipse.swt
 
 Main plug-in for the SWT user interface library.
 
+Development overview:
+-------------
+
+SWT is made up of 2 parts, Java and the natives (written in C).
+
+* **Java Part**:
+
+If you only make changes in the Java part or need to configure SWT for running
+snippets then you don't need to recompile the natives and you can just
+use the pre-compiled libraries in the binary repository instead.
+
+* **Natives Part**
+
+Make sure the binary project for your platform is imported and open in your workspace,
+_for example_
+  * `binaries/org.eclipse.swt.win32.win32.x86_64` for Windows or
+  * `binaries/org.eclipse.swt.gtk.linux.x86_64` for Linux or
+  * `binaries/org.eclipse.swt.cocoa.macosx.x86_64` for Mac
```

Comment:
Maybe it is worth to also make some note on `aarch64`, as currently these examples only refer to fragments for environments with x86 architecture but ARM is becoming more and more common.

Reply:
Mentioned in d556545966b1beefd598027b20675dfa34cde7e6